src/main/java/com/example/glovo/controller/OrderController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.glovo.controller;
+
+import com.example.glovo.dto.OrderDto;
+import com.example.glovo.service.OrderService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    private OrderService orderService;
+
+    @GetMapping("/{id}")
+    public OrderDto findById(@PathVariable int id) {
+        return orderService.getById(id);
+    }
+
+    @PostMapping()
+    public boolean save(@RequestBody OrderDto orderDto) {
+        return orderService.save(orderDto);
+    }
+
+    @PutMapping("/{id}")
+    public OrderDto update(@PathVariable int id, @RequestBody OrderDto orderDto) {
+        return orderService.update(id, orderDto);
+    }
+
+    @PatchMapping("/{id}/products")
+    public OrderDto updateProducts(@PathVariable int id, @RequestBody String product) {
+        return orderService.updateProducts(id, product);
+    }
+
+    @DeleteMapping("/{id}/products")
+    public OrderDto deleteProduct(@PathVariable int id, @RequestBody String product) {
```

Comment:
do not use the body of the delete request. any product identification is better

Reply:
but in my implementation - product is String, wich product identification i need to use?

Reply:
e.g DELETE /orders/{id}/products/{product-name} or DELETE  /orders/{id}/products/{product-id}